sphinx/setup_command.py

```diff
@@ -174,6 +176,8 @@ def run(self):
             confoverrides['today'] = self.today
         if self.copyright:
             confoverrides['copyright'] = self.copyright
+        if self.nitpicky:
+            confoverrides['nitpicky'] = self.nitpicky
```

Comment:
It seems mypy warns for this line.
https://travis-ci.org/sphinx-doc/sphinx/jobs/396257990

Could you add type hints to `# type: Dict[unicode, Any]` to L.168 please?